src/styles/blocks/card.scss

```
@@ -0,0 +1,89 @@
+.card {
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $color-aqua;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__img {
+    max-width: 160px;
+    height: 134px;
+    display: flex;
+    margin: 0 auto;
+  }
+
+  &__heading {
+    margin: 40px 0 0;
+    line-height: 18px;
+    font-weight: 500;
+    font-size: $fs-12;
+    color: $color-navy-dark;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: $fs-10;
+    line-height: 14px;
+    color: $color-grey;
+    margin: 4px 0 0;
+  }
+
+  &__rating {
+    margin: 16px 0 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-count {
+      font-weight: 400;
+      font-size: $fs-10;
+      line-height: 14px;
+      color: $color-navy-dark;
+    }
+  }
+
+  &__price {
+    margin: 24px 0 0;
+    display: flex;
+    justify-content: space-between;
+
+    &-text {
+      font-weight: 400;
+      font-size: $fs-12;
+      line-height: 18px;
+      color: $color-grey;
+    }
+
+    &-amount {
+      font-weight: 700;
+      font-size: $fs-basic;
+      line-height: 18px;
+      color: $color-navy-dark;
+    }
+  }
+
+  &__btn {
+    margin: 16px 0 0;
+    display: flex;
+    padding: 11.5px 55.5px 11.5px 54.5px;
+    background-color: $color-azure;
+    font-weight: 700;
+    font-size: $fs-14;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    border-radius: 5px;
+    color: $color-white;
+    border: 1px solid $color-white;
+    cursor: pointer;
+    transition: 0.4s;
+
+    &:hover {
+      color: $color-azure;
+      background-color: $color-white;
+      border: 1px solid $color-azure;
```

Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover
```suggestion
      border-color: $color-azure;
```